### ISSUE TITLE
React Native, Theme Dark, Unreadable text

### ISSUE THREAD

**Issue by TfADrama** [storybookjs/react-native#37]
**Describe the bug**
Changing the theme to dark with react native causes some storybook's welcome text to be unreadable...

**To Reproduce**
Change the theme to dark by adding this in your ./storybook/index.js

**Expected behavior**
Storybook server with readable text.

**Screenshots**
![image](https://user-images.githubusercontent.com/1742789/78402804-2666de80-75f3-11ea-89e1-d22dd3f920eb.png)

**Code snippets**
```
import { addParameters } from '@storybook/react-native';
import { themes } from '@storybook/theming';

addParameters({
  options: {
    theme: themes.dark,
  },
});

```

**System:**
 System:
    OS: macOS 10.15.2
    CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
  Binaries:
    Node: 12.12.0 - /usr/local/bin/node
    Yarn: 1.19.1 - /usr/local/bin/yarn
    npm: 6.13.6 - /usr/local/bin/npm
  Browsers:
    Chrome: 80.0.3987.149
    Firefox: 72.0.2
    Safari: 13.0.4
  npmPackages:
    @storybook/addon-actions: ^5.3.18 => 5.3.18 
    @storybook/addon-links: ^5.3.18 => 5.3.18 
    @storybook/addons: ^5.3.18 => 5.3.18 
    @storybook/react-native: ^5.3.18 => 5.3.18 
    @storybook/react-native-server: ^5.3.18 => 5.3.18 
    @storybook/theming: ^5.3.18 => 5.3.18 

**Additional context**

**Comment by dannyhw** [comment]
a good first issue, I'd say the first steps would be to reproduce this and find what exactly is hard to read and why.

**Comment by kiranjd** [comment]
Reproduced this setup. Currently, it looks like only the commands as in the description need a change of colour.
To be completely sure, will have to see the add-on tabs which is not possible because of issue #24 at the moment.

**Comment by dannyhw** [comment]
@kiranjd this issue with addons will be fixed soon, you can make it appear for now by changing app/react-native-server/src/client/manager/index.js as follows

```
setTimeout(() => {
    renderStorybookUI(rootEl, new Provider({ url: location.host, options: storybookOptions }));
}, 0);
```

**Comment by kiranjd** [comment]
Thanks, @dannyhw  that worked.
I have verified dark mode for actions, knobs and notes. They are fine.
I'll issue a PR with the above one fixed

**Comment by kiranjd** [comment]
Issued a PR #101 with a fix.

**Comment by dannyhw** [comment]
fixed in the 5.3.22 release thanks to #101 by @kiranjd
